### PR TITLE
Product listing

### PR DESCRIPTION
**Pull request by ritu9898**

### CODE REVIEW

File: pages/listing.html [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+        </div>
+        <div class="add-to-cart">
+          <i class="fa-solid fa-cart-shopping"></i>
+        </div>
+      </div>
+    </div>
+  </div>
+</div>
+<script>
```
Comment (sanu29) [inline pages/listing.html:32]: make a new file for js

**Review by sanu29 — APPROVED** [review]
good for now

File: pages/styles/listing.css [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+  transition-duration: 0s;
+}
+
+.is-active {
+  transition-duration: 1s;
+  background-position: -2800px 0;
+}
+
+.add-to-cart {
+  font-size: 30px;
```
Comment (developer-anand) [inline pages/styles/listing.css:69]: use rem here.

File: pages/styles/listing.css [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+#product-listing .child-card i.add-to-wishlist:hover {
+  cursor: pointer;
+}
+
+#product-listing .child-card i.add-to-wishlist:active {
+  color: red;
+}
+
+.heart {
+  width: 100px;
```
Comment (developer-anand) [inline pages/styles/listing.css:54]: rem ?

**Review by developer-anand — APPROVED** [review]
Be consistent with units use rem where ever possible. Good to go with changes.

**Pull request closed without merging**